lib/python/vdm/server/HTTPListener.py

```diff
@@ -1258,9 +1260,14 @@ def put(database_id, server_id):
         if 'force' in request.args:
             is_force = request.args.get('force').lower()
         else:
-            is_force = "false"
+            is_force = 'false'
 
-        if is_force == "true":
+        if 'stopserver' in request.args:
```

Comment:
why does stopserver needs a flag doesnt /server/server-id/stop stops the requested server?